bin/install-requirements.bash

```diff
@@ -7,9 +7,13 @@ export ANSIBLE_STDOUT_CALLBACK=yaml
 echo "This script will install all requirements"
 echo "Enter your password to start the install"
 
-if [ "${#}" -ge 2 ] && [ "${2}" = "--user" ]; then
+if [ "${#}" -ge 1 ] && [ "${1}" = "--user" ]; then
   export CK8S_INSTALL_PATH="${CK8S_INSTALL_PATH:-"${HOME}/bin"}"
   ansible-playbook -e 'ansible_python_interpreter=/usr/bin/python3' --ask-become-pass roles/get-requirements.yaml
 else
-  ansible-playbook -e 'ansible_python_interpreter=/usr/bin/python3' --ask-become-pass --become --become-user root roles/get-requirements.yaml
+  if [ "${#}" -ge 1 ] && [ "${1}" = "--become" ]; then
```

Comment:
What about `--no-ask-become-pass`? `--become` already has a meaning.

Reply:
changed this to `--no-pass` short and sweet

Reply:
Nice! :)